docs/root/configuration/http_filters/lua_filter.rst

```diff
@@ -290,6 +290,17 @@ under the filter name i.e. *envoy.lua*. Below is an example of a *metadata* in a
 
 Returns a :ref:`metadata object <config_http_filters_lua_metadata_wrapper>`.
 
+requestInfo()
+^^^^^^^^^^^^^
+
+.. code-block:: lua
+
+  requestInfo = handle:requestInfo()
+
+Returns some :repo:`information <include/request_info/request_info.h>` related to the current request.
```

Comment:
nit: remove "some" (we can add more later and people can read below about what is actually available)